### ISSUE TITLE
KiCad support?

### ISSUE THREAD

**Issue by David-Gould** [streamx3/stm32symutil#1]
This would be even more useful if it could generate symbols for KiCad.

**Comment by streamx3** [comment]
> This would be even more useful if it could generate symbols for KiCad.

Yes, I think it's a right thing to do, but I'm not familiar with KiCad.
If you could suggest a table/text format that can be easily imported as KiCad symbol -- please let me know.
I can see there's a loot of similar tools. I'll check uConfig, it looks like it could be the one we need.
https://github.com/xesscorp/kicad-3rd-party-tools
